modules/administration_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -320,9 +320,8 @@ First, go to the admin settings and set up the required WND mounts. Be
 aware though, that there are some limitations. These are:
 
 * We need access to the Windows account password for the mounts to
-update the file cache properly. This means that `__login credentials,
-saved in session__" won’t work with the listener. "__login credentials, saved in DB__`
-should work and could be the best replacement.
+update the file cache properly. This means that `__login credentials, saved in session__` won’t work with the listener.
+"__login credentials, saved in DB__" should work and could be the best replacement.
```

Comment:
why do we have on 
login credentials saved in session single backticks
and at 
login credentials saved in DB double quotes ?

Reply:
Honestly? No idea.